### PR TITLE
docs(sdk): Manually update chaneglog for master

### PR DESCRIPTION
**Pull request by WiNloSt**
[Slack discussion](https://metaboat.slack.com/archives/C063Q3F1HPF/p1732286276909939?thread_ts=1731680130.243949&cid=C063Q3F1HPF)

### Description

After we've redeployed version 0 for 1.52.x, we should fix the changelog accordingly to prevent confusion.

### CODE REVIEW

**Comment by npretto** [comment]
Should we also rename 1.51? https://github.com/metabase/metabase/pull/50447/files#diff-d36366ea0c0db75f1f528cae974feec799d37b91ce54c93a27881c85fb96b8c1L80

**Comment by WiNloSt** [comment]
@npretto fixed in b961e57d32d1c45bb5c42d351be5a0b4339201fe

File: enterprise/frontend/src/embedding-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## [0.52.2-nightly](https://github.com/metabase/metabase/compare/embedding-sdk-1.52.1...embedding-sdk-0.52.2-nightly) (2024-11-21)
+## [0.52.2-nightly](https://github.com/metabase/metabase/compare/embedding-sdk-0.52.1-nightly...embedding-sdk-0.52.2-nightly) (2024-11-21)
```
Comment (WiNloSt) [inline enterprise/frontend/src/embedding-sdk/CHANGELOG.md:1]: Old diff: https://github.com/metabase/metabase/compare/embedding-sdk-1.52.1...embedding-sdk-0.52.2-nightly
New diff: https://github.com/metabase/metabase/compare/embedding-sdk-0.52.1-nightly...embedding-sdk-0.52.2-nightly

File: enterprise/frontend/src/embedding-sdk/CHANGELOG.md [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 
 
-## [1.52.1](https://github.com/metabase/metabase/compare/embedding-sdk-0.1.38...embedding-sdk-1.52.1) (2024-11-07)
+## [0.52.1](https://github.com/metabase/metabase/compare/embedding-sdk-0.1.38...embedding-sdk-0.52.1-nightly) (2024-11-07)
```
Comment (WiNloSt) [inline enterprise/frontend/src/embedding-sdk/CHANGELOG.md:33]: Old diff: https://github.com/metabase/metabase/compare/embedding-sdk-0.1.38...embedding-sdk-1.52.1
new diff: https://github.com/metabase/metabase/compare/embedding-sdk-0.1.38...embedding-sdk-0.52.1-nightly

File: enterprise/frontend/src/embedding-sdk/CHANGELOG.md [hunk 3]
```diff
@@ -77,7 +77,7 @@
 
 
 
-# [1.51.0](https://github.com/metabase/metabase/compare/embedding-sdk-0.1.37...embedding-sdk-1.51.0) (2024-10-08)
+# [0.51.0](https://github.com/metabase/metabase/compare/embedding-sdk-0.1.37...embedding-sdk-0.51.0) (2024-10-08)
```
Comment (WiNloSt) [inline enterprise/frontend/src/embedding-sdk/CHANGELOG.md:80]: Old diff: https://github.com/metabase/metabase/compare/embedding-sdk-0.1.37...embedding-sdk-1.51.0
New diff: https://github.com/metabase/metabase/compare/embedding-sdk-0.1.37...embedding-sdk-0.51.0

**Review by npretto — APPROVED** [review]

**Review by heypoom — APPROVED** [review]
I visited each diffs and verified - works as expected 💯

**Pull request merged**